### PR TITLE
Add CNCF reference in the README

### PR DESCRIPTION
**Pull request by caniszczyk**
A simple reference to the CNCF and link to the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 Visit [prometheus.io](https://prometheus.io) for the full documentation,
 examples and guides.
 
-Prometheus is a systems and service monitoring system. It collects metrics
+Prometheus, a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics
```
Comment (brian-brazil) [inline README.md]: I'd put this down the end of this section.

Comment (fabxc) [inline README.md]: I think CI checks don't like the trailing `/` or something.

Comment (grobie) [inline README.md]: I guess the point of branding is to place it quite prominent. Given the support we get from CNCF, I'm personally fine having it here.

Comment (fabxc) [inline README.md]: Yea, agreed.

Comment (caniszczyk) [inline README.md]: I'm flexible, just figured it would be a nice place to put it.

I have no idea why Travis isn't happy

Comment (grobie) [inline README.md]: I fixed the broken test in master.

**Comment by grobie** [comment]
Thanks!

**Pull request merged**